_commands/caen/Caen-Connect-Tutorial.md

```diff
@@ -0,0 +1,149 @@
+
+<img src="http://mobileapps.its.umich.edu/sites/all/themes/umzentwo/images/umich-app-icon.png" alt="Umich" style="width: 100px; height: 100px;"><br>
+
+## Introduction
+
+**Caen-Connect-Tutorial** walks through with you the way to sync your local project folder to your remote CAEN (Linux) machine **instantly** using **SFTP**, so you could ssh into CAEN, and do the stuff that you want CAEN to do.
+
+## Possible Cases of Usage
+
+Possible scenarios of use including:
+
+- You are sitting in your dorm, developing your EECS484 project 01/02/03/04, and you want to develop **locally** using you favorite text editor (let's say [sublime text](https://www.sublimetext.com)) due to its reach functionality in auto-completion, syntax highlighting, auto-correction, auto-write-the-whole-project-so-I-could-just-sit-there-and-do-nothing, etc. But, sadly, you can only run your code on CAEN since only CAEN can run ***sqlplus***. You are too lazy to get out of the couch, go to duder/BBB and get a CAEN machine, and start working. But you have to get your code there. Shoot!
+- You are, again, writing your C++ code for EECS281/482/483/4xx projects **locally**. At the end of the spec, the instructor want you to run your code in this specific version of ***g++*** (let's say 4.8.5) on a Linux machine (possibly using some Linux system calls so a Mac/PC is simply not going to get you there). But you don't have Linux installed locally. Even if you do, you find that the **g++** version is 5.1.5 and there is no way for you to downgrade it. ONLY CAEN runs **g++** 4.8.5. Shoot!
+- You have a remote Linux machine that is so so so powerful, why installing my own Linux? But how do I get my code up there? Shoot!
+
+## Make It Happen
+
+Now, to make this happen, all you need is a [sublime text](https://www.sublimetext.com) editor (More text editor options are to come as soon as I figure them out).
+
+
+#### Some Preparations
+1. Go ahead to [www.sublimetext.com](https://www.sublimetext.com) to get the latest version of sublime text installed on your machine (Mac/PC/Linux). Note that it doesn't matter whether you install version 2 or 3. They all works. (Skip if you already have done it in the past.)
+2. Install **Package Control**. [Here](https://packagecontrol.io/installation)'s how you could do it, or you could google it by yourself. (Skip if you already have done it in the past.)
+3. Hit `Cmd-Shift-P` if you are working on a Mac (`Ctrl-Shift-P` on PC) to open up the prompt. Type `Install Package` and hit `Enter`. In the next prompt, type **`SFTP`** and hit `Enter`. The installation will perform automatically. Restart your editor after this.
+
+#### Real Magic
+1. **Configurations**
+
+	Open up your **root/parent project directory** (where you put all your project folders for this course) and create a new json file named **`sftp-config.json`**. Open it up, and copy-paste in the following code:
+
+	```json
+	{    
+	    "type": "sftp",
```

Comment:
For whatever reason, this doesn't render too well on the website, leaving the leading to this ...
![image](https://cloud.githubusercontent.com/assets/7095187/22857507/22e915fa-f074-11e6-8197-66de2d536d16.png)